School/app.rb

```diff
@@ -0,0 +1,106 @@
+require './teacher'
+require './student'
+require './book'
+require './rental'
+require './classroom'
+require './person'
```

Comment:
- [ ] [OPTIONAL] I would suggest that you use `require_relative` here because files are located in the same folder 😃 👍